edb/edgeql-parser/src/parser.rs

```diff
@@ -55,6 +55,7 @@ pub fn parse<'a>(input: &'a [Terminal], ctx: &'a Context) -> (Option<&'a CSTNode
     let mut new_parsers = Vec::with_capacity(parsers.len() + 5);
 
     for token in input {
+        // println!("token {:?}", token);
```

Comment:
Do you want to keep the debug code?

Reply:
Yes, ive used it a few times now.

Maybe i should have put it behind a feature flag instead of a comment.